internal/infoblox/infoblox.go

```diff
@@ -784,3 +803,20 @@ func lookupEnvAtoi(key string, fallback int) (i int) {
 	}
 	return
 }
+
+func deserializeEAs(extAttrJSON string) (map[string]interface{}, error) {
+	extAttrs := make(map[string]interface{})
+	if extAttrJSON == "" {
+		return extAttrs, nil
+	}
+	// Check if the input is stringified JSON
+	var intermediate string
+	if err := json.Unmarshal([]byte(extAttrJSON), &intermediate); err == nil {
+		extAttrJSON = intermediate
+	}
```

Comment:
thx for change, its more readable now. Isn't this section also redundant ? If the input is not valid, you can find out about it directly in the following section. My point is that an invalid input will throw the exact same error in the next section as it would in this section. If the entry is valid, then the next section will be valid too.

Reply:
You're right, this was a "fix" for a problem that doesn't exist but is caused by user error (myself included, I misconfigured my input JSON....)